components/combobox/combobox.jsx

```diff
@@ -132,6 +142,7 @@ const propTypes = {
 	 */
 	labels: PropTypes.shape({
 		label: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
+		labelRequired: PropTypes.boolean,
```

Comment:
Please be consistent with `Input` and use top level `required` prop.
https://github.com/salesforce/design-system-react/blob/master/components/forms/input/index.jsx#L223

Reply:
ah cool - yeah will do. That's one thought I had when adding that prop. Should have brought it up.